File: Consigna2.py
import random
numeros = []
pares = 0
impares = 0
for i in range(500):
    n = random.randint(50, 100)
    print("Numero generado", i + 1, ":", n)
    numeros.append(n)
    if n % 2 == 0:
        pares += 1
    else:
        impares += 1
print("Cantidad de números pares:", pares)
print("Cantidad de números impares:", impares)

File: Consigna2.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main(){
    srand(time(0)); 
    int numeros[500];
    int pares = 0;
    int impares = 0;
    for (int i = 0; i < 500; i++) {
        numeros[i] = rand() % (100 - 50 + 1) + 50; 
        printf("Número generado %d: %d\n", i + 1, numeros[i]);
        if (numeros[i] % 2 == 0) {
            pares++;
        } else {
            impares++;
        }
    }
    cout << "La cantidad de números pares es: " << pares << endl;
    cout << "La cantidad de números impares es: " << impares << endl;
    return 0;
}